README.md

```diff
@@ -33,3 +33,27 @@ Include diagrams in the site:
 * Update the relevant markdown file to incorporate diagrams into the site.
 * 
 * `git commit` changes
+
+### Run the application locally
+
+Running this application is fairly manual at the moment and requires some technical expertise.  In summary, you can run the application on your own computer by downloading it to your computer, setting up a few dependencies and executing a number of shell commands to start desired components. You will need on your machine:-
+* [postgres](https://www.postgresql.org/download/) installed on the target machine
+* [Docker](https://docs.docker.com/get-docker/)
+* [Node](https://nodejs.org/en/download/) to a GUI/front end application such as Plan Limits (which is a React application)
+* Java and [Gradle](https://gradle.org/install/)
```

Comment:
You don't need to install grade because of `gradlew`

Reply:
Good point.  Removed